### DISCUSSION TITLE
ROSCO installation tutorials

### DISCUSSION THREAD

**Discussion by Uisliu** [NatLabRockies/ROSCO#295] [Q&A]
I am currently using OpenFAST for a project to simulate a floating 15 MW turbine. I plan to utilize the precompiled version of OpenFAST since I don't have any experience with Fortran coding. Additionally, I intend to use the precompiled version of ROSCO to incorporate the 15 MW turbine.

Are these appropriate decisions for a basic simulation of this model?

I successfully downloaded the precompiled binaries version of OpenFAST, but I am encountering difficulties with the download of ROSCO. I find the tutorials a bit cryptic. Are there any good discussions that may assist me on this journey?

**Comment by dzalkind** [comment]
It sounds like you're on the right path.  You may need to learn a bit of python and Fortran along the way, but Google and ChatGPT should be sufficient.

If you have specific questions about the [standard workflow documentation](https://rosco.readthedocs.io/en/latest/source/standard_use.html), please add them here or search other discussions on this page.
